src/main/java/com/exasol/adapter/dialects/exasol/ExasolFromExaQueryRewriter.java

```diff
@@ -21,7 +24,11 @@ public ExasolFromExaQueryRewriter(final SqlDialect dialect, final RemoteMetadata
     }
 
     @Override
-    protected String generateImportStatement(String connectionDefinition, String pushdownQuery) throws SQLException {
-        return "IMPORT FROM EXA " + connectionDefinition + " STATEMENT '" + pushdownQuery.replace("'", "''") + "'";
+    protected String generateImportStatement(final String connectionDefinition, final String pushdownQuery)
+            throws SQLException {
+        final String importStatement = "IMPORT FROM EXA " + connectionDefinition + " STATEMENT '"
+                + pushdownQuery.replace("'", "''") + "'";
+        LOGGER.finer(() -> "IMPORT push-down statement:\n" + importStatement);
```

Comment:
You can also use EXPLAIN VIRTUAL to get this information

Reply:
I still find it useful to log it. The statements are also logged for the other connection types.